src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -115,14 +154,26 @@ def load_arguments(self, _):
             c.argument('item_name', item_name_type)
 
     with self.argument_context('backup protection backup-now') as c:
-        c.argument('retain_until', type=datetime_type, help='The date until which this backed up copy will be available for retrieval, in UTC (d-m-Y).')
+        c.argument('retain_until', type=datetime_type, help='The date until which this backed up copy will be available for retrieval, in UTC (d-m-Y).', options_list=['--retain-until', '-rt'])
+        c.argument('backup_type', arg_type=get_enum_type(['Full', 'Differential', 'Log', 'CopyOnlyFull']), options_list=['--backup-type', '-bt'])
```

Comment:
similarly, other shorthands?

Reply:
Most of them are. For those that are not will confirm.